lib/options.js

```diff
@@ -355,6 +380,10 @@ function findElmJsonPath(args, subcommand) {
   return findUp.sync('elm.json') || null;
 }
 
+/**
+ *
+ * @returns {(flag: string) => true}
```

Comment:
Not saying you need to change it, but this is a smell that this should return `void` instead, right? :sweat_smile:

Reply:
Oh yeah, for sure 😄